src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs

```diff
@@ -156,16 +161,54 @@ private void CreateWorkTime(Guid projectId, List<Guid> userIds, List<string> err
             }
         }
 
+        private List<Guid> CreateImage(List<CreateProjectImageRequest> projectImages, Guid userId, List<string> errors)
+        {
+            if (projectImages == null || projectImages.Count == 0)
+            {
+                return new();
```

Comment:
return null. Don't create object what we don't need

Reply:
done